types/googlemaps/index.d.ts

```diff
@@ -3373,8 +3376,8 @@ declare namespace google.maps {
         }
 
         enum RankBy {
-            DISTANCE,
-            PROMINENCE,
+            DISTANCE = 1,
+            PROMINENCE = 0,
```

Comment:
Wrong `DISTANCE` & `PROMINENCE` value.

![image](https://user-images.githubusercontent.com/10235949/67053373-e6e22e00-f149-11e9-8f5f-9af3321500d7.png)

Reply:
The values are correct, but yeah the order was wrong.